src/frontend/components/administration/update/UpdateProcessDialog.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <v-dialog :value="visible" max-width="1000px" @input="closeDialog">
+    <v-card>
+      <v-toolbar
+        color="primary"
+        dark
+      >
+        <span class="mr-2" v-text="$t('administration.update.title')" />
+      </v-toolbar>
+      <v-card-text>
+        <v-container fluid>
+          <v-select
+            v-model="interval"
+            :items="intervals"
+            :label="$t('administration.update.interval')"
+          />
+          <v-text-field
+            v-model="startDate"
+            :label="$t('administration.update.startDate')"
+            :rules="[dateFormatRule, dateIsFuturRule]"
+          />
+          <v-text-field
+            v-model="endDate"
+            :label="$t('administration.update.endDate')"
+            :rules="[dateFormatRule, dateIsFuturRule]"
+          />
+        </v-container>
+      </v-card-text>
+      <v-card-actions>
+        <v-btn
+          text
+          class="red--text"
+          @click.stop="closeDialog()"
+          v-text="$t('cancel')"
+        />
+        <v-spacer />
+        <v-btn
+          text
+          :disabled="!validForm"
+          class="green--text"
+          @click="startUpdate()"
+          v-text="$t('create')"
+        />
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script>
+export default {
+  name: 'UpdateDialog',
+  props: {
+    dialog: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data () {
+    return {
+      interval: 'day',
+      startDate: this.$dateFns.format(new Date(), 'yyyy-MM-dd'),
+      endDate: this.$dateFns.format(new Date(), 'yyyy-MM-dd'),
+      dateFormatRule: value => this.$dateFns.isMatch(value, 'yyyy-MM-dd') || 'YYYY-MM-DD',
+      dateIsFuturRule: value => Date.now() > new Date(value) || this.$t('administration.update.futur'),
+    }
+  },
+  computed: {
+    intervals () {
+      return ['day', 'week']
+    },
+    validForm () {
+      return this.$dateFns.isMatch(this.startDate, 'yyyy-MM-dd') &&
+        this.$dateFns.isMatch(this.endDate, 'yyyy-MM-dd') &&
+        new Date(this.endDate) <= Date.now()
+    },
```

Comment:
You can use a [v-form](https://vuetifyjs.com/en/components/forms/) if you need to keep track of the valid state without duplicating rules. Plus using a form allow for submitting by pressing `Enter` in a text field.